gitpod-io/gitpod
# [protocol, et al] Extract gRPC ClientCallMetrics into protocol

## Description

During work on #10325 it became apparent we had duplicate/non-reusable code in various places.
This PR moves it into a central place (`gitpod-protocol/src/util/grpc.ts`) and makes use of it across the code base.

## Related Issue(s)

Context: https://github.com/gitpod-io/gitpod/issues/10325

## How to test
 - start any workspace
 - see how it works as expected

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:
